app/views/ui-components/v1/cards/_metal_level_select.html.slim

```diff
@@ -755,7 +787,9 @@ javascript:
         metalLevel: "#{@benefit_package_form.sponsored_benefits.first.reference_product.metal_level_kind}",
         carrierName: "#{@benefit_package_form.sponsored_benefits.first.reference_product.issuer_name}",
         planType: "#{@benefit_package_form.sponsored_benefits.first.product_package_kind}",
-        referencePlanID: "#{@benefit_package_form.sponsored_benefits.first.reference_plan_id}"
+        referencePlanID: "#{@benefit_package_form.sponsored_benefits.first.reference_plan_id}",
+        isStandardPlan: "#{@benefit_package_form.sponsored_benefits.first.reference_product.is_standard_plan}",
```

Comment:
Similar to above: converting these booleans to strings with `isXXX` names may cause confusion around type for future maintainers. Keeping as booleans in the JS may be clearer.